R/router.R

```diff
@@ -119,18 +137,42 @@ create_router_callback <- function(root, routes) {
       }
     )
 
-    observeEvent(session$userData$shiny.router.page(), {
+    shiny::observeEvent(session$userData$shiny.router.page(), {
       page_path <- session$userData$shiny.router.page()$path
       log_msg("shiny.router main output. path: ", page_path)
-      shiny::removeUI("#router-page-content")
-      shiny::insertUI(
-        "#router-page-wrapper", "afterBegin",
-        shiny::div(id = "router-page-content", routes[[page_path]][["ui"]])
-      )
+      session$sendCustomMessage("switch-ui", page_path)
     })
   }
 }
 
+router_server <- function(router) {
```

Comment:
should this be exported, or users should use only `make_router`?

Reply:
This is internal function, and users should use only make_router.

Reply:
ok, but internal function should have short docs too, please